Formula/rust-mode.rb

```diff
@@ -3,6 +3,8 @@
 class RustMode < EmacsFormula
   desc "Emacs major mode for editing Rust code"
   homepage "https://github.com/rust-lang/rust-mode"
+  url "https://github.com/rust-lang/rust-mode/archive/0.3.0.tar.gz"
+  sha256 "6ec0c86eb9a19b850d6e4a42db56f80ff9586cc2013d8b2f9a652fc5f0e7be45"
   head "https://github.com/rust-lang/rust-mode.git"
 
   depends_on :emacs => "23.1"
```

Comment:
This should also fix the error on Travis (which apparently installs Emacs 23.3.1 by default):

You must install the `ert` dependency; see README.md